github.js

```diff
@@ -0,0 +1,40 @@
+// Intialise GitHub API
+const github = require("octonode");
+const client = github.client(process.env.GITHUB_TOKEN);
+const scopes = {
+    scopes: ["user", "repo"],
+    note: "admin script",
+};
+
+// GitHub API Methods
+const ghme = client.me();
+const ghsearch = client.search();
+
+// TODO: Refactor var content to variables - array? class? or just use content instead of client
+const ghuser = client.user("Inoxia25");
+const ghrepo = client.repo("MLH-Fellowship/pod-3.1.3-team-4");
+const ghissue = client.issue("MLH-Fellowship/pod-3.1.3-team-4", 4);
+const ghpr = client.pr("pksunkara/hub", 37);
+
+// Get PRs
+async function getPullRequests() {
+    const result = await ghrepo.prsAsync({ per_page: 100 });
+    return result[0];
```

Comment:
I'm impressed that you used `async` / `await`. Perhaps you could add some error-handling here in case `result` returns an empty array?